content/pt-br/kubernetes.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Kubernetes
+status: Completed
+category: tecnologia
+tags: ["infrastructure", "fundamental", ""]
+---
+
+## O que é
+
+Kubernetes, normalmente abreviado como K8s, é um orquestrador de containers de código aberto. 
+Ele automatiza o ciclo de vida de aplicações em containers em infraestruturas modernas, funciona como um "sistema operacional de datacenter" que gerencia aplicativos em um [sistema distribuído](/distributed-systems/).
+
+O Kubernetes planeja os [contêineres](/container/) entre [nós](/nodes/) de um [cluster](/cluster/), agrupando vários recursos de infraestrutura como balanceador de carga, armazenamento persistente, etc., para executar aplicações em containers.
+
+O Kubernetes permite automação e extensibilidade, permitindo que os usuários implantem aplicativos declarativamente (veja abaixo) de maneira reproduzível. É extensível por meio de sua [API](/application-programming-interface/), permitindo que profissionais experientes do Kubernetes aproveitem seus recursos de automação de acordo com suas necessidades.
```

Comment:
Redirect to the Portuguese page in the /application-programming-interface/ link by adding /pt-br/